<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WomenswearDialog</class>
 <widget class="QDialog" name="WomenswearDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 202, 222, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QCheckBox" name="skirtCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Skirt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="topCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QPushButton" name="woFinishButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bagCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bag</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="woLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="woNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Item:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bikiniCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Bikini</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
